pandas/tests/reductions/test_reductions.py

```diff
@@ -159,6 +159,14 @@ def test_same_tz_min_max_axis_1(self, op, expected_col):
         expected = df[expected_col].rename(None)
         tm.assert_series_equal(result, expected)
 
+    def test_numpy_reduction_with_tz_aware_dtype(self, tz_aware_fixture):
```

Comment:
can you parametrize and test both np.minimum & np.maximum;

Reply:
What's the benefit of that?